tools/editor/scene_tree_dock.cpp

```diff
@@ -1877,6 +1895,13 @@ void SceneTreeDock::_bind_methods() {
 	ObjectTypeDB::bind_method(_MD("instance"),&SceneTreeDock::instance);
 }
 
+SceneTreeDock::NodeNameCasing SceneTreeDock::_get_node_name_casing(const String& p_property) {
+	switch (EditorSettings::get_singleton()->get(p_property).operator int()) {
+		case 1: return NODE_NAME_CASING_CAMEL_CASE;
+		case 2: return NODE_NAME_CASING_SNAKE_CASE;
+	}
+	return NODE_NAME_CASING_PASCAL_CASE;
+}
```

Comment:
I guess I'll completely get rid of that method anyway as @akien-mga requested. Reason I did it at first was because it was based on the editor zoom mode which gets the setting with a similar method (I didn't know if that was a recommended way to handle enum settings).